.github/workflows/goreleaser.yaml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
```

Comment:
This file was being identified as both a github workflow and a goreleaser config file in vscode - this will remove invalid flagged errors when editing this file.